packages/common/src/utils/fileUtils.ts

```diff
@@ -35,3 +36,15 @@ export const getDownloadFilename = ({
     return `${split.join('.')}.mp3`
   }
 }
+
+export const dedupFilenames = (files: DownloadFile[]) => {
```

Comment:
calling out that (_I think_) if there are dupes, the first occurence won't have `-1` in it.
I don't have a problem with that though. Also, my filename dn db migration has same behavior lol, so kinda consistent that way (but I think inconsistent with how we've done it in the past where there is `1` for the first dupe... I think)

Reply:
oh. also. should the second dupe start from `-2` instead of `-1`?

Reply:
hm don't think it rly matters tbh, i was following mac os convention where they append a `(1)` to the first dup. and your migration only affects old stem files right?
i don't feel strongly tho!